crypto/bio/bio_lib.c

```diff
@@ -485,16 +485,6 @@ int BIO_recvmmsg(BIO *b, BIO_MSG *msg,
     return ret;
 }
 
-int BIO_get_rpoll_descriptor(BIO *b, BIO_POLL_DESCRIPTOR *desc)
-{
-    return BIO_ctrl(b, BIO_CTRL_GET_RPOLL_DESCRIPTOR, 0, desc);
```

Comment:
Should `BIO_CTRL_GET_RPOLL_DESCRIPTOR` be removed from bio.h.in (same for `_WPOLL_`)?

Reply:
yes, thanks, done.